### ISSUE TITLE
doubt

### ISSUE THREAD

**Issue by aaqqws** [khanhlee/snare-cnn#1]
May I ask if the last step in your data processing is to use the Sigmoid function for scaling? Theoretically, the final data is between 0 and 1. Why are there negative numbers in your training set pssm.cv.csv and test set pssm.ind.csv? Did I misunderstand this?
